filterprotobuffer_test.go

```diff
@@ -0,0 +1,25 @@
+package dapper_test
+
+import (
+	"github.com/dogmatiq/dapper/internal/fixtures"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"testing"
+)
+
+func TestPrinter_ProtobufFilter(t *testing.T) {
+	protoStub := &fixtures.Protostub{FirstField: "hello", EnumField: fixtures.Protoenum_FOO}
+	protoStub.ProtoReflect().SetUnknown(protoreflect.RawFields("\x12\x07testing"))
+	_ = protoStub.String()
+	test(
+		t,
+		"protobuffer type",
+		protoStub,
+		"*fixtures.Protostub<proto.Message>{",
+		`    first_field: "hello"`,
+		"    enum_field: 1",
+		"    unknownFields: {",
+		"        00000000  12 07 74 65 73 74 69 6e  67                       |..testing|",
+		"    }",
+		"}",
```

Comment:
> What the output format should look like. There's an example in the test assertion.

A few thoughts come to mind:

- I think it would be good to show the protocol buffers "tag" (the field number) in front of the field name
- If possible with `protoreflect`, we could then show the unknown fields in the same way (without the name, of course)
- Align the field values horizontally for consistency with existing output
- Render enum names as well as integer value
- It would probably be a good idea to test with a second protocol buffers message as a field of this one

```
*fixtures.Protostub{
    [ 1] first_field: "hello"
    [ 2] enum_field:  FOO (1)
    [27] <unknown>:   testing
}
```